Makefile

```diff
@@ -148,12 +149,15 @@ build: install build-$(CALYPSO_ENV)
 
 build-css: public/style.css public/style-rtl.css public/style-debug.css public/editor.css
 
-build-development: build-server $(CLIENT_CONFIG_FILE) server/devdocs/search-index.js build-css
+build-inline-js:
+	@$(UGLIFYJS) -o public/catch-js-errors.js catch-js-errors.js
```

Comment:
Even then, our current process supports building different chunks or we can make it build a separate file. The idea is to use the same process and tools like webpack, instead of using uglify on its own.